app/Services/GithubWebhookService.php

```diff
@@ -4,44 +4,39 @@
 
 namespace App\Services;
 
-use App\DTO\MemberDTO;
-use App\DTO\OrganizationDTO;
-use Illuminate\Support\Collection;
+use App\Models\Organization;
+use App\Models\User;
 
 class GithubWebhookService
 {
-    public function __construct(
-        protected OrganizationService $organizationService,
-    ) {}
-
-    public function handle(Collection $data): void
-    {
-        switch ($this->getActionType($data)) {
-            case "created":
-                $organizationData = $data["installation"]["account"];
-
-                if ($organizationData["type"] === config("services.organization.type")) {
-                    $organization = OrganizationDTO::createFromArray($organizationData);
+    public function createOrganization(
+        string $organizationName,
+        int $organizationId,
+        string $organizationAvatarUrl,
+    ): void {
+        Organization::create([
+            "name" => $organizationName,
+            "github_id" => $organizationId,
+            "avatar_url" => $organizationAvatarUrl,
+        ]);
+    }
 
-                    $this->organizationService->create($organization);
-                }
+    public function removeMember(
+        int $organizationId,
+        int $memberId,
+    ): void {
+        $user = User::query()->where("github_id", $memberId)->first();
 
-                break;
-            case "member_removed":
-                $organization = OrganizationDTO::createFromArray($data["organization"]);
-                $member = MemberDTO::createFromArray($data["membership"]["user"]);
+        if (!$user) {
+            throw new \InvalidArgumentException("User with GitHub ID {$memberId} not found.");
```

Comment:
Is it good exception? Maybe create new one?

Reply:
changed it, now catching and throwing ModelNotFoundException